lib/main.js

```diff
@@ -52,9 +51,13 @@ function multiple(selections, stake, ew = false, fullCover = false) {
   return allCombos;
 }
 
+function effectiveOdds(selections) {
+  return selections.reduce((acc, selection) => acc * toDecimal(selection.odds), 1);
```

Comment:
Do you think it would be clearer if we moved this return to a descriptive const and returned that?

```
const itDoesThis = CODE
return itDoesThis;
```

Reply:
I think this one is a matter of taste, and my taste is not to do that. However if the general opinion is to do it then I will graciously concede

Reply:
If we're to consider the options for that value:

It returns the `effectiveOdds`, which is the name of the method which is one line above,

Or `productOfSelectionOdds` which kind of describes the operation in the body.

I'm easy too though